Upstream: not yet, waiting for changes of kfilemetadata to be applied upstream

From 4a2a83384a1610771a956dcd24b55044710e719f Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Fri, 23 May 2014 12:33:34 +0200
Subject: [PATCH] Adapt to KFileMetaData becoming a framework

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a07aa4..0a92171 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,8 +5,8 @@ find_package(ECM 0.0.12 REQUIRED NO_MODULE)
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules" ${ECM_MODULE_PATH})
 
 find_package(Qt5 REQUIRED NO_MODULE COMPONENTS Widgets DBus Test Sql)
-find_package(KF5 REQUIRED COMPONENTS I18n Config Service IdleTime KCMUtils Auth Runner Crash Solid KIO KDELibs4Support)
-find_package(KFileMetaData 4.97 REQUIRED)
+find_package(KF5 REQUIRED COMPONENTS I18n Config Service IdleTime KCMUtils Auth Runner Crash Solid KIO Auth KDELibs4Support)
+find_package(KF5 REQUIRED FileMetaData)
 
 include(ECMSetupVersion)
 include(ECMAddTests)
-- 
2.0.0
